products/health-care/appointments/va-online-scheduling/research/july-2020-user-research/vaosr-june-research-plan.md

```diff
@@ -1,50 +1,86 @@
+     - Users are looking for a specific provider, but can't find the specific provider they have in mind
+   - Users are confused by the different look/feel, if this is new to them. Easily give up, rather than trying to figure it out 	
+
 ## Method	
 1.	What method of research are you planning? 	
-  - **Sample Methodologies**: Directed interviews; ethnographic interviews; observation; card sorts; tree tests; guerilla usability testing; usability testing with low- or high-fidelity prototypes; user acceptance testing. 	
-  - **Also mention:** in-person moderated, remote moderated, remote unmoderated	
-  	
-2.	Why this method? How does this methodology help you answer your research questions? 	
+   - Remote moderated usability research and usability testing mid-fidelity prototypes
+2.	Why this method? How does this methodology help you answer your research questions? 
+   -  
 3.	Where are you planning to do your research? *If in person, mention the location, point of contact, arrangements, etc. If online, mention which tool you'll be using (GTM, Join.me, etc.)*	
-4.	What will you be testing? *(Design mocks, card sort, prototype, page, content, etc.)* 	
-5.  If remote: What tool do you plan to use (Zoom, GoToMeeting, Webex)	
+   - Zoom
+4.	What will you be testing? *(Design mocks, card sort, prototype, page, content, etc.)* 
+   - https://www.va.gov/health-care/schedule-view-va-appointments/
+5.	If remote: What tool do you plan to use (Zoom, GoToMeeting, Webex)	
+   - Zoom
+
 ## Participants and Recruitment	
-1.	Participant criteria: What are you looking for in a participant?	
-(Mention: Number of people, ages, accessibility preferences, geographical diversity, login requirements, VA benefit requirements, familiarity with technology, etc. Keep in mind, the more requirements, the more difficult the recruit, so give ample time to ensure the right participant mix.)	
-2.	What is your recruitment strategy? 	
-(If in person, describe how you will find participants. If remote, mention if you plan to draw from the existing recruiting contract or if there are other places where you would like to reach out to find participants specifically for this project. If you need help, please contact UX lead.)	
+1. Participant criteria: What are you looking for in a participant?	
+   - 20	
+   - Ideal, but not required: individuals who need to or who have recently scheduled an appointment
+
+2. What is your recruitment strategy? 	
+   Perigean	
+
 ## When? 	
 1.	Timeline: What dates do you plan to do research? 	
 (IF you are using the research recruiting contract, please submit 1 FULL week prior to the start of research for remote, 2+ weeks for in person.) 	
 2.	Prepare: When will the thing you are testing be ready? (Goes without saying, but should be a few days before testing will begin.) 	
```

Comment:
The tool is ready. We do need Ryan to update the V2 of the EC mock with content updates. They should be done sometime next week when he's back from vaca.